src/coreclr/jit/targetloongarch64.h

```diff
@@ -0,0 +1,332 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#pragma once
+
+#if !defined(TARGET_LOONGARCH64)
+#error The file should not be included for this platform.
+#endif
+
+// NOTE for LoongArch64:
+//  The `REG_R21` which alias `REG_X0` is specially reserved !!!
```

Comment:
```
//  The `REG_R21` which alias `REG_X0` is specially reserved !!!
//  It can be used only by manully and should be very careful!!!
```
=>
```
//  `REG_R21` which aliases `REG_X0` is specially reserved !!!
//  It can be used only manually and very carefully!!!
```

Reply:
Ok, Thanks.